app/src/pages/common/modals/importModal/constants.js

```diff
@@ -0,0 +1,16 @@
+export const MODAL_TYPE_IMPORT = 'import';
+export const MODAL_TYPE_UPLOAD = 'upload';
+
+export const ACCEPT_FILE_MIME_TYPES = {
+  [MODAL_TYPE_IMPORT]: [
```

Comment:
I suggest to rename this constants to provide more information about their destination.
F.e. `MODAL_TYPE_IMPORT` is about launches and `MODAL_TYPE_UPLOAD` about plugins, so it can be `MODAL_TYPE_IMPORT_LAUNCH` and `MODAL_TYPE_UPLOAD_PLUGIN` respectively.